components/script/dom/webidls/Node.webidl

```diff
@@ -30,8 +30,10 @@ interface Node : EventTarget {
 
   [Pure]
   readonly attribute Document? ownerDocument;
+
   [Pure]
-  readonly attribute Node rootNode;
+  Node getRootNode(optional GetRootNodeOptions options);
```

Comment:
I would prefer if you declared this as `Node getRootNode();`, without the argument. That makes it more obvious that the argument is not implemented yet.